profiles/desc/cpu_flags_arm.desc

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+aes - Use the AES cryptography instruction set
+crc32 - Use the CRC32 instruction set
+edsp - Use the enhanced DSP instructions (ARMv*E and ARMv6+)
+iwmmxt - Use the iwMMXt instruction set
+neon - Use the NEON (ASIMD) instruction set
+sha1 - Use the SHA-1 cryptography instruction set
+sha2 - Use the SHA-2 cryptography instruction set
+thumb - Enable Thumb instruction set (ARMv*T and ARMv6+)
+thumb2 - Enable Thumb-2 instruction set (ARMv*T2 and ARMv7+)
+v4 - Use ARMv4 instructions
+v6 - Use ARMv6 instructions
+v7 - Use ARMv7 instructions
+v8 - Use ARMv8 instructions
+vfp - Use the VFP instruction set
+vfp-d16 - Indicate that the FPU has 16 64-bit VFP registers
+vfp-d32 - Indicate that the FPU has 32 64-bit VFP registers
+vfpv3 - Use the VFP version 3 instruction set
```

Comment:
Well, how kernel and arm exposes this is one thing, what's most convenient for us is another one. They probably have reasons to expose it that way, but in that case I don't get them.

The way I understand it, the following required use would always hold:

vfpv3? ( ^^ ( vfp-d16 vfp-d32 ) )
vfp-d16? ( vfpv3 )
vfp-d32? ( vfpv3 )


So, in order to avoid having to write this required use line in every ebuild, to me it makes more sense to have:

vfpv3 - vfpv3 with at least 16 double registers
vfpv3-d32 - vfpv3 with at least 32 double registers

This should make all the cases expressible but avoids the need for required use.